src/mongocrypt-ctx-encrypt.c

```diff
@@ -2471,6 +2471,14 @@ _check_cmd_for_auto_encrypt(mongocrypt_binary_t *cmd, bool *bypass, char **colln
         *bypass = true;
     } else if (0 == strcmp(cmd_name, "getLog")) {
         *bypass = true;
+    } else if (0 == strcmp(cmd_name, "createSearchIndex")) {
```

Comment:
```suggestion
    } else if (0 == strcmp(cmd_name, "createSearchIndexes")) {
```

To follow naming described in DRIVERS-2549.